angular2-jwt.ts

```diff
@@ -1,6 +1,6 @@
 import {Injectable, Injector} from 'angular2/core';
 import {Http, HTTP_PROVIDERS, Headers, BaseRequestOptions, Request, RequestOptions, RequestOptionsArgs, RequestMethod, Response} from 'angular2/http';
-import {Observable} from 'rxjs/Observable';
+import {Observable} from 'rxjs';
```

Comment:
rxjs does a lot of weird stuff with their exports. You essentially have to read the code to be sure of what it's doing.